src/civic_canister_frontend/src/App.tsx

```diff
@@ -63,26 +63,18 @@ function App() {
   );
 }
 
-const id = ["id", { Text: "did:example:c276e12ec21ebfeb1f712ebc6f1" }]
-const name = ["name", { Text: "Example University" }]
-const degreeType = ["degreeType", { Text: "MBA" }]
-// Example Credential with mixed claims
-const alumniOfClaim = {
-  claims: [id, name, degreeType]
-}
-
 const mixedClaim = {
   claims: [
-    ["Is over 18", { Boolean: true }],
-    ["name", { Text: "Max Mustermann" }],
-    ["alumniOf", { Claim: alumniOfClaim }]
+    ["passType", { Text: "tigoYhp9SpCDoCQmXGj2im5xa3mnjR1zuXrpCJ5ZRmi" }],
+    ["status", { Text: "active" }],
+    ["expirationDate", { Number: 1000 + 365 * 24 * 60 * 60 * 1000 }]
```

Comment:
elsewhere (e.g. in your demo vid) this field was a string or date type. Which is correct?

Reply:
This has been updated. It should be a string. Changes were not committed.